velox/docs/monthly-updates/february-2023.rst

```diff
@@ -0,0 +1,77 @@
+********************
+February 2023 Update
+********************
+
+
+Core Library
+============
+
+ * Add several memory related fixes: task reference in partition output iobuf to prevent memory leaking,
+   memory corruption caused by ntile.
+ * Add concurrent memory allocation benchmark.
+ * Add various enhancements such as replace ArrayBuilder class with ArrayWriter, support for decimal types in DecodedVector,
```

Comment:
Split into separate items. Drop some.

* Add support for DECIMAL types to DecodedVector.

>  BloomFilter to support serialization and memory tracking

What is this about? @Yuhta can you help with wording?